app/src/androidTest/java/com/example/bigowlapp/fragments/ScheduleFormFragmentTest.java

```diff
@@ -241,6 +243,23 @@ public void completeAScheduleForm() {
 
         // Click on Set Schedule
         confirmSetSchedule.perform(click());
+
+        //Verify Dialog
+        StringBuilder selectedMembers = new StringBuilder();
+        for (User user : fakeListOfUsers) {
+            selectedMembers.append(user.getFullName()).append("\n");
+        }
+        String startDate = dateFormatter(startNewTime);
+        String startTime = timeFormatter(startNewTime);
+        String endDate = dateFormatter(newEndTime);
+        String endTime = timeFormatter(newEndTime);
+        String selectedlocation = (fakeSelectedLocation.placeName() == null) ? location.address() : location.placeName();
+
+        onView(withText("Please confirm the new Schedule info: \n\nTitle:\n" + "Schedule_Title" + "\n\nGroup name:\n"+ "group_name" + "\n\nSelected members:\n"+ selectedMembers
+                + "\nStart date:\n" + startDate + "\n"+ startTime + "\n\nEnd date\n" + endDate + "\n" + endTime + "\n\nSelected Location:\n" + selectedlocation))
+                .check(matches(isDisplayed()));
+        onView(withText("CONFIRM")).perform(click());
```

Comment:
Nice job integrating your additions to the test, both fixing it and increasing coverage a bit